docs/http-clients.md

```diff
@@ -0,0 +1,17 @@
+# HTTP Clients
+
+To date, we have four different HTTP API clients:
+
+- [kubo-rpc-client](https://github.com/ipfs/js-kubo-rpc-client) - Official JS client for talking to Kubo over HTTP
+- [ipfs-http-client](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs-http-client) - http client for js-ipfs -- Use when talking to [`js-ipfs`](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs)
```

Comment:
My vote would be that we don't even bring this up.

Reply:
This was in the docs previously. I'm down to remove them but didn't want to take anything out that already existed without input from kubo team

Reply:
Could defer this to https://github.com/ipfs/ipfs-docs/issues/1194

Reply:
If we're adding in discussion about kubo-rpc-client, I wouldn't try to explain ipfs-http-client.  Lets just simplify the message and talk about the new thing.

Reply:
ok sounds good. It looks like this PR was based on an out-of-date commit somehow. https://github.com/ipfs/kubo/pull/9331/files#diff-b3523638f13db6204ab4b8011784399cf53055844ae4e5411a1d420767b6cb81